### PR TITLE
[CONTRIBUTING] explain CHANGELOG syntax & integration specs update process

### PR DESCRIPTION
**Pull request by jpsim**
Hopefully this clarifies some questions that potential contributors would have. Should also address #316.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
+When issuing a pull request, please add a summary of your changes to
+the `CHANGELOG.md` file.
+
+We follow the same syntax as CocoaPods' CHANGELOG.md:
+
+1. One Markdown unnumbered list item desribing the change.
+2. 2 trailing spaces on the last line describing the change.
+3. A list of Markdown hyperlinks to the contributors to the change. One entry
+   per line. Usually just one.
+4. A list of Markdown hyperlinks to the issues the change addresses. One entry
```
Comment (pcantrell) [inline CONTRIBUTING.md]: But not PRs, right, just issues? Worth mentioning.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+git checkout master
+git pull
+git checkout -
+git rebase master
+bundle install
+bundle exec rake rebuild_integration_fixtures
+cd spec/integration_specs
+git checkout -b $jazzy_branch_name
+git commit -a -m "update for $jazzy_branch_name"
+git push
```
Comment (pcantrell) [inline CONTRIBUTING.md]: This requires write access to specs repo, doesn't it?

**Comment by pcantrell** [comment]
Great addition. Wish I’d had this when I first started contributing! Certainly will be helpful to others.

You’re on fire today!

**Comment by jpsim** [comment]
Thanks for the feedback. Incorporated.

**Pull request merged**